pipeline/relation_embedder/relation_embedder/src/test/scala/uk/ac/wellcome/relation_embedder/ArchiveRelationsCacheTest.scala

```diff
@@ -27,25 +27,35 @@ class ArchiveRelationsCacheTest
   val work3 = work("a/3")
   val work4 = work("a/4")
 
+  val relA = Relation(workA, depth = 0, numChildren = 4, numDescendents = 9)
+  val rel1 = Relation(work1, depth = 1, numChildren = 2, numDescendents = 2)
+  val relB = Relation(workB, depth = 2, numChildren = 0, numDescendents = 0)
+  val relC = Relation(workC, depth = 2, numChildren = 0, numDescendents = 0)
+  val rel2 = Relation(work2, depth = 1, numChildren = 2, numDescendents = 3)
+  val relD = Relation(workD, depth = 2, numChildren = 0, numDescendents = 0)
+  val relE = Relation(workE, depth = 2, numChildren = 1, numDescendents = 1)
+  val relF = Relation(workF, depth = 3, numChildren = 0, numDescendents = 0)
+  val rel3 = Relation(work3, depth = 1, numChildren = 0, numDescendents = 0)
+  val rel4 = Relation(work4, depth = 1, numChildren = 0, numDescendents = 0)
```

Comment:
nit: sorting these would make it easier to follow

Reply:
Yes it is weird, I just followed the sorting which existed already for the works which I'm sure I originally sorted according to some specific but indecipherable rules :see_no_evil: